components/layout.tsx

```diff
@@ -44,16 +47,18 @@ export const Layout: React.FC<React.PropsWithChildren<LayoutProps>> = ({
             <title>{formattedTitle}</title>
             <link rel="icon" href="/favicon.ico" />
           </Head>
-          <PageContainer>
-            {isMobile ? <MobileNav /> : <DesktopNav />}
-            <AuthModal />
-            <div className={`col`}>{children}</div>
-            <PageFooter
-              authenticated={authenticated}
-              user={user as any}
-              signOut={signOutAndRedirectToHome}
-            />
-          </PageContainer>
+          <FollowContext.Provider value={{ followStatus, setFollowStatus }}>
```

Comment:
We maintain a list of providers in `components/providers.tsx` that keeps a central point of access for all of our core context providers - it might make sense to add this as a Provider there to help keep our providers organized